_bep_collection/bep043.md

```diff
@@ -0,0 +1,4 @@
+---
+redirect_to:
+  - https://docs.google.com/document/d/1LACjc5hFXDpa2l_QddBPR41Vce_gglGv9WeuBB7LsBU/edit
```

Comment:
@effigies What is the difference between the link above and this link:

[https://docs.google.com/document/d/1LACjc5hFXDpa2l_QddBPR41Vce_gglGv9WeuBB7LsBU/edit?usp=sharing](https://docs.google.com/document/d/1LACjc5hFXDpa2l_QddBPR41Vce_gglGv9WeuBB7LsBU/edit?usp=sharing)

That's something I've never understood. Like, does the `?usp=sharing` at the end actually do anything for people?

Reply:
It's just a tag for Google where the link was created. https://stackoverflow.com/questions/30624321/google-drive-links-url-format-and-query-parameters